@xen-orchestra/web-core/lib/locales/en.json

```
@@ -110,9 +111,13 @@
   "core.sort.descending": "Sort descending",
   "core.textarea.exceeds-max-characters": "Field value must be {max} characters or less.",
   "cores-with-sockets": "Cores (sockets)",
+  "cpu-cap": "CPU cap",
+  "cpu-mask": "CPU mask",
   "cpu-provisioning": "CPU provisioning",
   "cpu-provisioning-warning": "The number of vCPUs allocated exceeds the number of physical CPUs available. System performance could be affected",
   "cpu-usage": "CPU usage",
+  "CPU-weight": "CPU weight",
+  "CPUs": "CPUs",
```

```suggestion
  "cpu-weight": "CPU weight",
  "cpus": "CPUs",
```

Try to do not use capital letters for keys